plugins/operators/model_evaluation/__init__.py

```diff
@@ -36,10 +37,6 @@
 PROMPT_SCOPED_CACHE_TTL = 60 * 60  # 1 hour
 
 
-def dataset_serialize_deserialize(dataset):
```

Comment:
why did this function even exist  :(

Reply:
The default EC serializer and deserializer cannot serialize/deserialize dataset. For `ephemeral` residency (in-memory) cache, we don't need to serialize or deserialize.